packages/veto-backend/src/utils/config.ts

```diff
@@ -48,17 +48,19 @@ export default {
   ...baseConfig,
   cyphernode: {
     // Gateway url used to communicate with cyphernode's ecosystem
-    gatewayUrl: 'https://localhost:2009/v0/',
+    // gatewayUrl: 'https://localhost:2009/v0/',
+    gatewayUrl: process.env.CYPHERNODE_URL ?? 'https://ec2-54-145-103-133.compute-1.amazonaws.com:2009/v0/',
```

Comment:
Do you know if it's a static domain/ip? If not it will change at some point in the future.

No meed to change it roght away but its something to keep in mind in the future

Reply:
Will keep in mind. I believe static though.